types/zapier-platform-core/index.d.ts

```diff
@@ -8,6 +8,9 @@
 
 import * as http from "http";
 
+export const createAppHandler: Function;
```

Comment:
Do you know what the parameters and return type should be?

Reply:
No, I didn't dig into this, but still thought that it was better to add it as a generic function. Would you prefer that I remove it?

Reply:
The `Function` type fails the lint, so replace it with `(...args: any[]) => any` at least.

Reply:
Thanks a lot for pointing out the solution. You're right that it was a bit lazy to just use a generic function, so I tried to specify them better, but must admit, that they're only slightly more specific in the updated pull request.